source/_docs/redis.md

```diff
@@ -36,8 +36,12 @@ For detailed information, see [Installing Redis on WordPress](/docs/wordpress-re
 
 1. Enable the Redis cache server from your Pantheon Site Dashboard by going to **Settings** > **Add Ons** > **Add**.
 
-2. Add the [Redis](https://www.drupal.org/project/redis) module from Drupal.org.
-
+2. Install and activate the [Redis](https://www.drupal.org/project/redis) module from Drupal.org.
```

Comment:
Suggest adding the Terminus command to install and enable the module for D8 and D7 while we're here